ansible/dimensiondata/dimensiondata_backup.py

```diff
@@ -38,8 +38,11 @@
   region:
     description:
       - The target region.
-    choices: %s
-    default: na
+    choices:
+      - Regions are defined in Apache libcloud project file = libcloud/common/dimensiondata.py 
+      - See https://libcloud.readthedocs.io/en/latest/compute/drivers/dimensiondata.html
+      - Note: Values avail in array dd_regions(). 
+    default: na aka North America
```

Comment:
This can confuse the reader on what the actual default value is. Please remove here and in other places. They can read link you added for more info.